pymc3/parallel_sampling.py

```diff
@@ -369,14 +370,13 @@ def __init__(
 
         self._progress = None
         self._divergences = 0
+        self._total_draws = 0
         self._desc = "Sampling {0._chains:d} chains, {0._divergences:,d} divergences"
         self._chains = chains
-        if progressbar:
-            self._progress = tqdm(
-                total=chains * (draws + tune),
-                unit="draws",
-                desc=self._desc.format(self)
-            )
+        self._progress = progress_bar(
+            range(chains * (draws + tune)), display=progressbar, auto_update=False
```

Comment:
oooh, this `display=progressbar` is fancy! I would move the import to the top for sure, then!